gatsby-node.js

```diff
@@ -34,7 +34,39 @@ exports.createPages = ({ graphql, boundActionCreators }) => {
           },
         })
       })
-      resolve();
     })
-  });
+    .then(() =>
```

Comment:
so for all the new pages we need to send queries for, we just chain them here?

Reply:
Yeah for dynamically generated pages the request needs to be in the gatsby-node.js file so we can just chain them here

Reply:
saweet